test/src/k-provider/ovp/provider.spec.js

```diff
@@ -83,6 +83,28 @@ describe('OVPProvider.partnerId:1082342', function () {
     );
   });
 
+  it('should be able to get media config by referenceId', done => {
+    provider = new OVPProvider({partnerId: partnerId, uiConfId: 38621471}, playerVersion);
+    sinon.stub(MultiRequestBuilder.prototype, 'execute').callsFake(function () {
+      return new Promise(resolve => {
+        resolve({response: new MultiRequestResult(BE_DATA.EntryWithUIConfNoDrmData.response)});
+      });
+    });
+    provider.getMediaConfig({referenceId: '1_rsrdfext'}).then(
+      mediaConfig => {
+        try {
+          mediaConfig.should.deep.equal(MEDIA_CONFIG_DATA.WithPluginsNoDrm);
```

Comment:
What does this test check?

Reply:
That it is valid. before the changes it would not pass

Reply:
I agree its not much of a test... Do have any other test I can add?

Reply:
I would check the referenceId passed on the request

Reply:
done

Reply:
@yairans please re-review